apps/music/js/endpoint.js

```diff
@@ -511,4 +515,16 @@ function open(blob) {
   });
 }
 
+function enableNFC(enabled) {
+  // This function is a no-op until we enable NFC once.
+  if (!nfcEverEnabled && !enabled) {
```

Comment:
For consistency with the queue and remote, we should instead have a function called `loadNFCShare` that follows the same pattern as `loadQueueSettings()` and `loadRemote()` where it sets a global variable called `nfcShare` to the `NFCShare` global once lazy loaded. Then in this function you can just do `loadNFCShare().then(() => nfcShare.enabled = enabled);`.